help/scim.md

```diff
@@ -89,6 +89,56 @@ Zulip's SCIM integration has the following limitations:
        Zulip account to be updated accordingly.
      * Unassigning a user from the app will deactivate their Zulip account.
 
+{tab|entraid}
+
+{!upgrade-to-plus-if-needed.md!}
+
+1.  Contact [support@zulip.com](mailto:support@zulip.com) to request the
+    **Secret Token** that Entra will use to authenticate to your SCIM API.
+
+1. Sign in to the Sign in to the [Microsoft Entra admin center](https://entra.microsoft.com/).
+
+1. Go to **Identity** -> **Applications** -> **Enterprise applications**.
+
+1. Select **New application** -> **Create your own application**.
+
+1. Complete the form:
+    * Enter a name for your application.
+    * Select the option **Integrate any other application you don't find in the gallery**.
+    * Click **Add** to create the new app. It will be added to your **Enterprise applications**.
+
+1. Continue to the app's management screen and click **Provisioning** in the left panel.
+
+1. In the **Provisioning Mode** menu, select **Automatic**  and specify the following fields:
+    * **Tenant URL**: `http://yourorganization.zulipchat.com/scim/v2/?aadOptscim062020`.
+      The `?aadOptscim062020` part of it is a feature flag that needs to be added to ensure
+      SCIM compliance by Entra ID.
```

Comment:
wat.

We may want to link to https://learn.microsoft.com/en-us/entra/identity/app-provisioning/application-provisioning-config-problem-scim-compatibility ?  Or put that in a comment / commit message somewhere?

It sounds like it was _intended_ to bevome unnecessary at some point.

Reply:
Yeah, maybe it's still scheduled to happen sometime soon, but I verified it's still needed or it sends booleans as a string `"true"` to the SCIM API. Funnily, their doc also claims `Note this feature flag currently does not work with on-demand provisioning` but it seemed to work fine in my testing.

I added a link to the microsoft doc page there and re-pushed. This `?aadOptscim062020` feels weird enough that it might warrant a sentence of justification right there in the instructions. This is how Akamai documents it at https://techdocs.akamai.com/mfa/docs/prov-users-azure-scim:
> In Tenant URL, enter the Base URL that you copied from the ​Akamai MFA​ provisioning page and append the following feature flag to your URL to ensure SCIM compliance: ?aadOptscim062020.